src/pudl/output/pudltabl.py

```diff
@@ -145,21 +145,15 @@ def pu_eia860(self, update=False):
             )
         return self._dfs["pu_eia"]
 
-    def pu_ferc1(self, update=False):
+    def pu_ferc1(self) -> pd.DataFrame:
         """Pull a dataframe of FERC plant-utility associations.
 
-        Args:
-            update (bool): If true, re-calculate the output dataframe, even if
-                a cached version exists.
-
         Returns:
-            pandas.DataFrame: a denormalized table for interactive use.
+            A denormalized table for interactive use.
         """
-        if update or self._dfs["pu_ferc1"] is None:
-            self._dfs["pu_ferc1"] = pudl.output.ferc1.plants_utils_ferc1(
-                self.pudl_engine
-            )
-        return self._dfs["pu_ferc1"]
+        return pd.read_sql_table("denorm_plants_utils_ferc1", self.pudl_engine).pipe(
+            apply_pudl_dtypes, group="ferc1"
```

Comment:
Do we have the option of using `enforce_schema()` here rather than `apply_dtypes()`? @e-belfer cataloged a number of places where they result in different data types (which is expected).

Reply:
Yes! All of the tables `PudlTabl` produces will have metadata so we can use `enforce_schema()`. I can make the change in this branch if Ella hasn't already done it in #2490.

Reply:
I decided not to add enforce_schema to all of the PudlTable methods given @zschira is working on a more general solution in #2498.